### PR TITLE
Remove direct Microsoft.NETCore.Platforms dependency.

### PR DESCRIPTION
**Pull request by NTaylorMullen**
- Microsoft.NETCore.App now pulls this package in.
- New Xunit runner pulls this in for desktop tests.

aspnet/Coherence-Signed#344

/cc @bricelam @natemcmaster

### CODE REVIEW

**Comment by natemcmaster** [comment]
:watch: on build passes. 
:shipit: when it does.

File: test/Microsoft.EntityFrameworkCore.CrossStore.FunctionalTests/project.json [hunk 1]
```diff
@@ -3,7 +3,6 @@
     "warningsAsErrors": true
   },
   "dependencies": {
-    "Microsoft.NETCore.Platforms": "1.0.1-*",
     "Microsoft.EntityFrameworkCore.InMemory.FunctionalTests": "1.0.0",
```
Comment (NTaylorMullen) [inline test/Microsoft.EntityFrameworkCore.CrossStore.FunctionalTests/project.json]: These guys pull it in transitively now via dotnet-test-xunit.

**Comment by NTaylorMullen** [comment]
@natemcmaster is the appveyor failure expected?

**Comment by natemcmaster** [comment]
This is https://github.com/aspnet/EntityFramework/issues/5711

**Comment by natemcmaster** [comment]
The test failure is unrelated to the changes to 🚢 :it:

**Comment by NTaylorMullen** [comment]
This is in.

**Pull request closed without merging**